### ISSUE TITLE
`prefers-reduced-motion`

### ISSUE THREAD

**Issue by solarrust** [doka-guide/content#2989]
В доке можете рассказать про то, для чего нужно это медиа-выражение, разобрать его значения, привести пример кода (будет круто, если сделаете ещё и демку).

Ссылки для старта:

- https://w3c.github.io/csswg-drafts/mediaqueries-5/#prefers-reduced-motion
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion
- https://web.dev/prefers-reduced-motion/
- https://www.smashingmagazine.com/2021/10/respecting-users-motion-preferences/
- https://tatiana-fokina-blog.ru/posts/css-media-features-for-a11y/
- https://doka.guide/css/media/

Полезные ссылки по тексту и демкам в Доке:

- [Руководство по стилю](https://github.com/doka-guide/content/blob/main/docs/styleguide.md).
- [Демки](https://github.com/doka-guide/content/blob/main/docs/demos/index.md).
- [Про стиль демок](https://github.com/doka-guide/content/blob/main/docs/demos/style.md).
- [Подпись автора](https://github.com/doka-guide/content/blob/main/docs/people.md).

Для этой доки уже готов плейсхолдер. Это просто черновик материала. Можете дополнить его или переписать всё с нуля 🐶

**Comment by katkopikat** [comment]
Я беру

**Comment by TatianaFokina** [comment]
@katkopikat, привет! 👋 Собираешься писать материал? Может быть нужна наша помощь?

**Comment by katkopikat** [comment]
@TatianaFokina привет! 
Да, собираюсь! Я написала где-то больше половины, нужно ещё добавить материал и  оформить. 
Простите, что долго, постараюсь на след. неделе ускорится и дописать 🙂

**Comment by katkopikat** [comment]
@TatianaFokina привет! 👋
Я в целом дописала статью. 

Хочу посмотерть локально, что получилось. Запустила проект по инструкции: главная страница запустилась, поиск работает, но остальные страницы не открываются из-за секьюрности.

Мб есть совет, что здесь можно и лучше сделать?)

![image](https://github.com/doka-guide/content/assets/69801699/1768c1aa-6fc7-4555-8b5f-94cfb1681714)

**Comment by katkopikat** [comment]
Отлчино, спасибо, Тань, тогда через pr и посмотрю!)

Да, к сожалению, другие страницы тоже не открываются, везде `cannot GET /` и далее роут.

Главная страница отображается до этого момента (скрин), карточки статей под этим блоком уже не рендерятся, если вбивать что-то в поиске, то подсказки появляются, но при переходе по любоый из них ошибка `cannot ....`.
В консоли при запуске докера никаких ошибок нет.

![image](https://github.com/doka-guide/content/assets/69801699/bc9d6887-02c3-4782-b01a-602c663827fd)
